compiler/src/dotty/tools/dotc/util/Positions.scala

```diff
@@ -166,8 +170,9 @@ object Positions {
   }
 
   /** An index coordinate */
-  implicit def indexCoord(n: Int): Coord = new Coord(n + 1)
-  implicit def positionCoord(pos: Position): Coord =
+  def indexCoord(n: Int): Coord = new Coord(n + 1)
+
+  def pos2coord(pos: Position): Coord =
```

Comment:
This commit LGTM

Reply:
Overall I am skeptical about merging this.